src/Routers/AggregateRouter.js

```diff
@@ -72,7 +58,48 @@ export class AggregateRouter extends ClassesRouter {
     });
   }
 
-  transformStage(stageName, stage) {
+  /* Builds a pipeline from the body. Originally the body could be passed as a single object,
+   * and now we support many options
+   *
+   * Array
+   *
+   * body: [{
+   *   group: { objectId: '$name' },
+   * }]
+   *
+   * Object
+   *
+   * body: {
+   *   group: { objectId: '$name' },
+   * }
+   *
+   *
+   * Pipeline Operator with an Array or an Object
+   *
+   * body: {
+   *   pipeline: {
+   *     group: { objectId: '$name' },
+   *   }
+   * }
+   *
+   */
+  static getPipeline(body) {
```

Comment:
can we get some unit tests on this one? This would be very nice :)

Reply:
Also if you can update the changelog, that would help me not do it :P

Reply:
Added